jenkins_jobs/modules/parameters.py

```diff
@@ -1171,4 +1387,19 @@ def gen_xml(self, xml_parent, data):
             if pdefs is None:
                 pdefs = XML.SubElement(pdefp, "parameterDefinitions")
             for param in parameters:
+                # Pass job name to the uno-choice plugin
+                if next(iter(param)) == "active-choices":
+                    param["active-choices"]["_project-name"] = data.get(
+                        "name", None
```

Comment:
I would not pass None by default here - a job must have a name, otherwise it should be a hard error